db/migrations/20221206085946_create_orders_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    order_number INT NOT NULL,
+    status_id INT NOT NULL,
+    price DECIMAL NOT NULL,
```

Comment:
```suggestion
    price DECIMAL(10, 2) NOT NULL,
```
예시와 같이 decimal field의 자릿수도 지정해주세요. 위 suggestion은 예시고 실제 지정하실 자릿값은 기획에 맞게 하시면 됩니다~